### ISSUE TITLE
Request: post on Curseforge.com, No core mod to download for anarchy???

### ISSUE THREAD

**Issue by Gunner76th** [asiekierka/FoamFix#24]
Can you please post your mod over to the Curseforge.com site? It will seriously help your mod gain attention, as well as take care of the site looking good for you.

**Comment by asiekierka** [comment]
The quote you brought up says "contains a coremod", not " requires a coremod". Contains implies that it's part of the package, the same JAR.

There are currently no plans to upload the mod to CurseForge or other mod hosting services besides the official website.

**Comment by asiekierka** [comment]
Also, that is not the definition of "coremod". (I prefer the term "library mod" for a mod which many other mods depend on.) A "coremod", in layman's terms, is a piece of special code loaded by Forge Mod Loader before any other mods (or, in fact, even Minecraft itself) which gets special capabilities, such as arbitrary modification of code as it gets loaded.

>"2) foam fix anarchy edition IS a coremod in and of itself. "

If we were to get fully technical, Anarchy is *both* a coremod and a regular mod - it contains both in one JAR and you can verify this by going to Forge's mod list in-game.
